providers/kannel/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@novu/kannel",
+  "version": "0.11.0",
+  "description": "A kannel wrapper for novu",
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "private": false,
+  "repository": "https://github.com/novuhq/novu",
+  "license": "MIT",
+  "keywords": [],
+  "scripts": {
+    "prebuild": "rimraf build",
+    "build": "run-p build:*",
+    "build:main": "tsc -p tsconfig.json",
+    "build:module": "tsc -p tsconfig.module.json",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "fix:lint": "eslint src --ext .ts --fix",
+    "test": "run-s build test:*",
+    "test:lint": "eslint src --ext .ts",
+    "test:unit": "jest src",
+    "watch:build": "tsc -p tsconfig.json -w",
+    "watch:test": "jest src --watch",
+    "reset-hard": "git clean -dfx && git reset --hard && yarn",
+    "prepare-release": "run-s reset-hard test"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=10"
+  },
+  "dependencies": {
+    "@novu/stateless": "^0.9.0",
```

Comment:
This should match the provider version and also Novu's current version. Right now is 0.12.0. I tried to push the changes needed to get your PR updated to 0.12.0 and solve conflicts but I had no write permissions in your repository. 😓

Reply:
Thanks for the review, I've changed the version